partner-manager/modules/ROOT/pages/_partials/tutorial/supplier/td-target-channel-aaz.adoc

```diff
@@ -1,9 +1,9 @@
 // Create Target Channel
 
-On the Target Channel side of the <<img-td-new-aaz>>, click *Create New Channel*.
+. On the Navigation Menu, Click Target Channel <<img-td-new-gearco>>, and then click *Create New Channel*.
```

Comment:
<<img-td-new-gearco>> will return the caption for that image; so, in the old text, it makes sense, but it doesn't really here - do you know what I mean? I can go ahead and accept it if you want and we can work it from there.

Reply:
Yeah I guess we can remove it. we can add it later if required.